### DISCUSSION TITLE
[Feature]: Provide prop "ExternalContentId" for notification center IOS only

### DISCUSSION THREAD

**Discussion by tiplefbxx** [TheWidlarzGroup/react-native-video#4975] [Ideas]
### What problem are you trying to solve?

It's a pretty nice that we can override metadata on notification center, but ios can easily find correct metada for media by using "ExternalContentIdentifier" as specific string that should match apple feeds
https://help.apple.com/itc/tvpumcstyleguide/#/itc0c92df7c9

It simplify working with metadata and also as main feature it populates your media progress to apple NowPlaying queue.

### Proposed solution

Provide a prop `ExternalContentId` / `ExternalContentIdentifier` to the iOS Notification Center, so iOS can find the correct metadata for media by matching the Apple feeds.

### Platforms

iOS

### Additional context

Apple style guide reference: https://help.apple.com/itc/tvpumcstyleguide/#/itc0c92df7c9

**Comment by TWG-AI** [comment]
Thanks for raising this — the team will consider it. Flagged as a potential good first issue for contributors interested in picking it up.

> I'm an AI assistant — if you're interested in contributing, check the CONTRIBUTING guide for setup steps.
